src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.data.tag;
+
+import java.util.Objects;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Logs the addition/removal of tags during a session 
+ * Association class of Person and Tag
```

Comment:
Good, but a more precise description is `Represents an adding or removing a Tag to/from a Person`.